src/jquery.daterangepicker.js

```diff
@@ -1978,17 +1978,43 @@
             var range;
             var startDate = opt.startDate ? moment(opt.startDate).add(!opt.singleMonth && month === 'month2' ? 1 : 0, 'month') : false;
             var endDate = opt.endDate ? moment(opt.endDate).add(!opt.singleMonth && month === 'month1' ? -1 : 0, 'month') : false;
+            var shortcuts = opt.shortcuts.prev || opt.shortcuts['prev-days'] ? true : false;

```

Comment:
`opt.shortcuts` might be undefined, causing an exception when accessing `opt.shortcuts.prev` or `opt.shortcuts['prev-days']`. Please run `npm run dev` locally and test that the plugin works as indented, with no exceptions.

![image](https://user-images.githubusercontent.com/1837605/45925142-84a00500-bf18-11e8-93ee-e42688117aa4.png)